docker-compose-local-testing.yml

```diff
@@ -0,0 +1,32 @@
+version: "3"
+
+services:
+  postgres:
+    hostname: postgres
+    image: postgres:14.4-alpine
+    volumes:
+      - postgres-volume:/var/lib/postgresql/data
+    networks:
+      - codecovapi
+    ports:
+      - "5436:5432"
+    environment:
```

Comment:
For the environment I would set `POSTGRES_HOST_AUTH_METHOD=trust` instead of using `POSTGRES_USER` and `POSTGRES_PASSWORD`